content/_data/changelogs/weekly.yml

```diff
@@ -10814,6 +10814,61 @@
   # pull: 5398 (PR title: Fix Java11 telemetry test broken after end of its sendings (MissingClassTelemetryTest))
   # pull: 5399 (PR title: Merge back security tests into corresponding main test classes)
 
+- version: '2.289'
+  date: 2021-04-20
+  changes:
+  - type: major bug
+    category: regression
+    pull: 5411
+    issue: 65327
+    authors:
+    - basil
+    message: |-
+      File parameters again correctly overwrite previous files (regression in 2.278).
+  - type: bug
+    category: bug
+    pull: 5405
+    issue: 53462
+    authors:
+    - daniel-beck
+    message: |-
+      Stop using deprecated untrusted submit events on modern browsers for compatibility with future Firefox releases.
+  - type: rfe
+    category: rfe
+    pull: 5424
+    authors:
+    - daniel-beck
+    message: |-
+      Improve UI of slow trigger administrative monitor.
+  - type: rfe
+    category: rfe
+    pull: 5412
+    issue: 65308
+    authors:
+    - res0nance
+    message: |-
+      Improve performance when creating or deleting nodes by reducing queue-lock contention.
+  - type: rfe
+    category: localization
+    pull: 5410
+    authors:
+    - takashiharano
+    message: |-
+      Update Japanese translations for login screen.
+
+  # pull: 5403 (PR title: Ignore jetty 10.x and jetty 11.x updates)
+  # pull: 5413 (PR title: Bump spring-security-bom from 5.4.5 to 5.4.6)
```

Comment:
IIUC it is important enough as a dependency update

Reply:
Included in 72a9eed5cb01922625bdbd30844657a0864fa4ff